src/tests/compiletest.md

```diff
@@ -78,7 +78,7 @@ The following test suites are available, with links for more information:
 
 ### Rustdoc test suites
 
-See [Rustdoc tests](../rustdoc.md#tests) for more details.
+See [Rustdoc § Tests](../rustdoc.md#tests) for more details.
```

Comment:
The rest of the dev guide doesn't use the section sign at all. It's a bit odd to deviate from the status quo

Reply:
How else should I differentiate between "The page titled 'rustdoc tests'" and "The 'tests' section of the 'rustdoc' page"?